### ISSUE TITLE
.org Plans: Receipt sometimes not sent / emailed for plan purchases

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#203]
There's an issue with emails being sent out. Here are the scenarios I tried out:

Upgrading from $99 plan to $299 plan. This worked - the email was sent for both purchases. However (!) it didn't work the second time I tried it with the same account.

I have signed up for both the $99 and $299 with a test account and with different sites, but never received an email except for the first two emails (from # 1 above)

**Comment by designsimply** [comment]
Tested and found that I received an email titled "Set Up Your Premium Services" for both the 1st and 2nd purchase (got it twice for the 2nd purchase), but I only received the email titled "Your New Akismet + VaultPress Bundle" for the 1st purchase and not the 2nd.

My testing steps were:

1. Go to `wp-admin` in your Jetpack blog
1. Click Jetpack in the left menu
1. Click Compare Options (near bottom right of the page)
1. Click Upgrade Now button under Premium
1. Fill out credit card details
1. Click Pay button
1. Check to see what emails have arrived

<strong>Result:</strong> expected a "Set Up Your Premium Services" and a "Your New Akismet + VaultPress Bundle" email for each purchase, but I did not receive the "Your New Akismet + VaultPress Bundle" email for the 2nd purchase

<strong>Video:</strong> [3m54s](https://cloudup.com/c57NVRgl2-2)

Here are screenshots of the emails for reference:

<strong>Set Up Your Premium Services</strong><br>(got this for both purchases, got a duplicate for the 2nd purchase)

<img alt="screen shot 2016-04-29 at 16 32 24" src="https://cloud.githubusercontent.com/assets/1119271/14931534/80659d3a-0e29-11e6-97b3-b9f4da4a19e8.png" width="400" />

<strong>Your New Akismet + VaultPress Bundle</strong><br>(only got this for the first purchase)

<img alt="screen shot 2016-04-29 at 16 31 58" src="https://cloud.githubusercontent.com/assets/1119271/14931538/93459f4a-0e29-11e6-8eed-7f08a758320a.png" width="400" />
